system/Database/Forge.php

```diff
@@ -556,6 +562,33 @@ public function createTable(string $table, bool $if_not_exists = false, array $a
 
 	//--------------------------------------------------------------------
 
+	/**
+	 * Post - Drop Table statements to be run
+	 *
+	 * @param string  $table         Table name
+	 * @return array
+	 */
+	protected function _postDropTableStatements(string $table)
+	{
+		  return array();
```

Comment:
can use short array syntax

Reply:
Hi there @samsonasik  I am still little new to all this, what is short array syntax ? Should I make it to rahter be return []; ? Is that what you mean ?

Reply:
yes; `return [];`